Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'scientist', '~> 1.0.0'
 gem 'flipper'
 gem 'flipper-active_record'
 gem 'flipper-ui'
+gem 'panoptes-client', github: "zooniverse/panoptes-client.rb"
```

Comment:
nothing per se - just haven't cut a release on this and tested the other apps. It should be fine though.